### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by tmeschter**
1. Include a high-level description of the repository.
2. Add basic information on the AsyncPackage and
   Microsoft.CodeAnalysis.Analyzers projects.
3. Update the Getting Started section.

### CODE REVIEW

**Comment by tmeschter** [comment]
@srivatsn @mavasani @zoepetard @jepetty @daking2014 Please review.

**Comment by srivatsn** [comment]
:+1:

File: README.md [hunk 1]
```diff
@@ -7,18 +14,31 @@ Debug | Release
...
+
+The AsyncPackage analyzer enforces good practices when writing code that makes use of C#'s `async` and `await` language features.
+
 MetaCompilation
 ---------------
-The MetaCompilation Analyzer is an analyzer that functions as a tutorial to teach users how to write an analyzer. It uses diagnostics and code 
-fixes to guide the user through the various steps required to create a simple analyzer. It is designed for a novice analyzer programmer with some previous programming
-experience.
+
+*Created by summer 2015 interns [Zoë Petard](https://github.com/zoepetard), [Jessica Petty](https://github.com/jepetty), and [Dan King](https://github.com/daking2014)*
```
Comment (daking2014) [inline README.md]: @tmeschter "Daniel King" please, thanks.

Comment (tmeschter) [inline README.md]: @daking2014 Will do!

**Pull request merged**